leetcode/generate_md.py

```diff
@@ -0,0 +1,22 @@
+fin = open('source_leetcode_data.txt', 'r')
```

Comment:
Check result:

* PR title must contain group number in ['1021', '1022', '1013', '2021', '2022']